lib/addRemove.js

```diff
@@ -64,7 +64,7 @@ function addAsync(version, useNow) {
 			let versionDir = nvsUse.getVersionDir(version);
 			removeDirectoryRecursive(versionDir);
 
-			let remoteUri = settings.remotes[version.remoteName];
+			let remoteUri = version.uri;
 			return downloadAndExtractAsync(version, remoteUri).then(() => {
```

Comment:
`version.uri` is different from `settings.remotes[version.remoteName]`. But it looks like the `remoteUri` parameter to `downloadAndExtractAsync()` is not even used anyway. At one point I refactored to use the `version.uri` instead, and I guess I forgot to remove that parameter. Can you remove it instead of changing unused code?